apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/CollectionAssertionSamples.kt

```diff
@@ -0,0 +1,83 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class CollectionAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
+    private val isGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+
+    @Test
+    fun isEmpty() {
+        expect(listOf<Int>()) toBe empty
+
+        fails {
+            expect(listOf(1, 2, 3)) toBe empty
+        }
+    }
+
+    @Test
+    fun isNotEmpty() {
+        expect(listOf(1, 2, 3)) notToBe empty
+
+        fails {
+            expect(listOf<Int>()) notToBe empty
+        }
+    }
+
+    @Test
+    fun hasSize() {
+        expect(listOf(1, 2, 3)) hasSize 3
+
+        fails {
+            expect(listOf(1, 2, 3)) hasSize 1
+        }
+    }
+
+    @Test
+    fun sizeFeature() {
+        expect(listOf(1, 2, 3)) size {
+            it isGreaterThan 1 // subject is now of type Int (actually 3)
+        } size {
+            it isLessThan 4 // subject is still of type Int (still 3)
+        }
+
+        fails {
+            expect(listOf(1, 2, 3)) size {
+                it isLessThan 1 // fails
+            } size {
+                it isGreaterThan 4 // not reported because `isLessThan(1)` already fails
+            }
+        } message {
+            contains("${isLessThanDescr}: 1")
+            containsNot("${isGreaterThanDescr}: 4")
+        }
+    }
+
+    @Test
+    fun size() {
+        expect(listOf(1, 2, 3)) size { // subject inside this block is of type Int (actually 3)
+            it isGreaterThan 1
+            it isLessThan 4
+        }
```

Comment:
please include the rest of the sample as defined in api-fluent use the same workaround with `/* */` as shown above

Reply:
I'm not sure what are You asking for. Can You take a look at the updated tests?

This is the overloaded size:
```
expect(listOf(1, 2, 3)) { // subject is now of type Int (actually 3)
    size isGreaterThan 1 isLessThan 4
}
```
This is the other:
```
expect(listOf(1, 2, 3)) size {
    isGreaterThan(1) // subject inside this block is of type Int (actually 3)
} size {
    isLessThan(4) // subject inside this block is of type Int (actually 3)
}
```

Reply:
See the suggestion I have made for sizeFeature. The problem with your current implementation is, that the comment is wrong. The subject inside the block is still List. It changes after the `size` call. See my suggestion, it looks a bit bewildered with the `/* */` but its (currently) the only way to add comments per line within the infix syntax. Let me know in case this explanation is not enough.

Reply:
What I meant with use the /**/ workaround: your sample is missing the note about, subject is back to list. You can do the following:
```
} /* Here comes the comment
*/ size {
...
}
```